modules/programs/terraform.nix

```
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.terraform;
+
+in {
+  meta.maintainers = [ maintainers.bryanhonof ];
```

```suggestion
  meta.maintainers = [ hm.maintainers.bryanhonof ];
```

HM's maintainers are in `lib.hm.maintainers` instead of Nixpkgs' `lib.maintainers`.